channeldb/payment_control.go

```diff
@@ -223,6 +223,18 @@ func (p *PaymentControl) InitPayment(paymentHash lntypes.Hash,
 	return updateErr
 }
 
+// DeleteFailedAttempts deletes all failed htlcs for a payment.
```

Comment:
nit: maybe update docs saying this would only happen if the db config is set?

Reply:
Done.